plugins/ua_network_pubsub_mqtt.c

```diff
@@ -189,6 +189,22 @@ UA_PubSubChannelMQTT_unregist(UA_PubSubChannel *channel, UA_ExtensionObject *tra
          return UA_STATUSCODE_BADARGUMENTSMISSING;
     }
 }
+/**
+ * receive a message.
+ *
+ * @return UA_STATUSCODE_GOOD if success
+ */
+static UA_StatusCode
+UA_PubSubChannelMQTT_receive(UA_PubSubChannel *channel, UA_ByteString *buffer,
+                             UA_ExtensionObject *transportSettings, UA_UInt32 timeout) {
+
+    /*TODO: add processing of the received messages to address space*/
+    UA_StatusCode ret = UA_STATUSCODE_GOOD;
```

Comment:
Return UA_STATUSCODE_BADNOTIMPLEMENTED instead. For whot purpose ist the following statement there 'buffer->length = 0;'? Idea -> Lets make the user-callback outputs here.